### ISSUE TITLE
June End-to-End doc review

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode-docs#374]
For June we want to have a doc review of the following sections:

- Overview
- Setup
- Editor
- Customization.

Read the documentation https://code.visualstudio.com/docs/
- file tasks for new features that should be covered in the doc
- file tasks for suggestions for improvements

### Zurich
- [x] @bpasero 
- [x] @weinand 
- [x] @isidorn 
- [x] @alexandrudima 
- [x] @jrieken 
- [x] @sandy081 
- [x] @dbaeumer 
- @egamma
- @aeschli (vacation)
- @joaomoreno 

### Redmond
- @kieferrm 
- @Tyriar 
- @chrmarti 
- @rebornix

**Comment by bpasero** [comment]
I reviewed up to (not including) "Languages" and also looked into "Runtime: node.js". I mainly filed issues around the fact that we introduced new UI that is not covered in the docs yet, specifically:
* tabs
* opened editors
* problems panel
* integrated terminal
* replace across files

**Comment by sandy081** [comment]
Reviewed till Customization and in build languages support. Filed issues for new functionalities, configurations and key bindings.

**Comment by jrieken** [comment]
I went for `Extensions` and `Extensibility Reference`

**Comment by isidorn** [comment]
I went for overview, setup, editor, customization. Filed [quite some issue](https://github.com/Microsoft/vscode-docs/issues/created_by/isidorn) all over the place, nothing too critical just that some of the docs need update with regards to the new features we introduced.

I could file more issue but I feel like it would be better if I spend more time on reading docs once @gregvanl reacts to the first round of feedback and the site gets updated to the newest release.
